br/pkg/lightning/backend/local/local.go

```diff
@@ -1129,6 +1129,8 @@ func (local *Backend) prepareAndSendJob(
 		needSplit = true
 	})
 	logger := log.FromContext(ctx).With(zap.String("uuid", engine.ID())).Begin(zap.InfoLevel, "split and scatter ranges")
+	// we are going to do max backoff of 15 min 30 second
+	backOffTime := 30 * time.Second
```

Comment:
init backoff is too large for transient error, and there's no max-backoff

Reply:
i added the max backoff. BR can suspend split for 60 to 300 seconds

Reply:
I also suggest to lower the initial backoff time. If there's some transient error like network problems, sleeping 30 seconds is too long. Maybe starting from 1 second.

Reply:
Lets start with 10 seconds. There is already a 1 second back off during region split and scatter.